axelrod/strategy_transformers.py

```diff
@@ -294,9 +318,15 @@ def final_sequence(player, opponent, action, seq):
         return seq[-index]
     return action
 
+def final_reclassifier(original_classifier, seq):
```

Comment:
We probably need to update the memory depth as well (also for initial transformer)